### ISSUE TITLE
Can't install private theme via Settings

### ISSUE THREAD

**Issue by zebrastribe** [afragen/github-updater#858]
Hi
I get this Message:
Downloader installationspakken fra https://api.github.com/repos/zebrastribe/hm/zipball/master…
Download mislykkedes. Not Found

I do not have a field for the github token key?
Where do I add it?

![image](https://user-images.githubusercontent.com/6557899/74733046-d0f58e80-524b-11ea-9961-45822dab9fa1.png)

**Comment by afragen** [comment]
Check the Bypass WP-Cron Background Processing. It's possible that you have an invalid access token or that you are having issues with WP-Cron.

Also, you should try the `develop` branch. There are authentication updates that I haven't released yet.

**Comment by zebrastribe** [comment]
Thank you so much. When I enable the "Bypass WP-Cron Background Processing for Debugging" the github field showed up and I can paste the token in. The themes which is hosted on gihub also show up also the 'Github Updater'. And I swichted to 'develop' and unchecked the "Bypass ..." field
(also I refreshed the cache)

But there is no new version update of the theme. 
(I have updated the version number in style.css)

I am guessing that the cron job needs to finish before the theme can be updated?
and that I properly have an issue the the cron job. Howto solve it?
